src/test/java/com/puppycrawl/tools/checkstyle/api/BeforeExecutionFileFilterSetTest.java

```diff
@@ -115,4 +116,14 @@ public void testUnmodifiableSet() {
             () -> filterSet.getBeforeExecutionFileFilters().add(filter));
     }
 
+    /*
+      This comment is not related to this code. Input based test does not call
```

Comment:
`This comment is not related to this code.`

Remove this

Reply:
@romani Done. 
i was getting PMD violations for this PR locally. Turns out PMD wouldn't allow whatever test I had written previously. 
![image](https://user-images.githubusercontent.com/90137881/227286600-fbfafbe9-c336-43e6-a740-b8ed4b79240c.png)
I have fixed it though.

Reply:
@romani Done.